lora_diffusion/dataset.py

```diff
@@ -326,8 +328,12 @@ def __getitem__(self, index):
         if self.use_template:
             assert self.token_map is not None
             input_tok = list(self.token_map.values())[0]
-
-            text = random.choice(self.templates).format(input_tok)
+            if self.custom_prompts:
```

Comment:
should we be more explicit? 

e.g.:
[{"image":"image_00.png", "prompt":"a photo of {}, being held by a human hand"}]

Reply:
Also, having the modulo means that there's no verification that the list of prompts and the list of images are the same.

Reply:
This assumes we're setting use_template to true, which isn't the case since we're specifying that we aren't using a template.